modules/terragrunt/generate.go

```diff
@@ -0,0 +1,38 @@
+package terragrunt
+
+import (
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/gruntwork-io/terratest/modules/testing"
+)
+
+// TgStackGenerate calls terragrunt stack generate and returns stdout/stderr
+func TgStackGenerate(t testing.TestingT, options *Options) string {
+	out, err := TgStackGenerateE(t, options)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return out
+}
+
+// TgStackGenerateE calls terragrunt stack generate and returns stdout/stderr
+func TgStackGenerateE(t testing.TestingT, options *Options) (string, error) {
+	if options.TerragruntBinary != "terragrunt" {
```

Comment:
Not sure if this is a valid check, can be `options.TerragruntBinary = /opt/v6.6.6/terragrunt`, will still be terragrunt and will fail the check

I think, since this is the terragrunt package, we should assume that the executable is terragrunt each time